test/ngIntegration/util/caseDetailsPage.js

```diff
@@ -0,0 +1,88 @@
+const BrowserWaits = require('../../e2e/support/customWaits');
+
+class CaseDetailsPage{
+
+    constructor(){
+        this.ccdCaseDetailsContainer = $('exui-case-details-home');
+        this.exuiAlert = $('exui-alert');
+
+        this.caseTitle = $('.title');
+        this.eventTriggerContainer = $('ccd-event-trigger');
+
+
+    }
+
+    async amOnPage(){
+        try{
+            await this.waitForPage();
+            return true;
+        }catch(err){
+            console.log("Case details page is not displayed.", err);
+            return false;
+        }
+    }
+
+    async waitForPage(){
+        await BrowserWaits.waitForElement(this.ccdCaseDetailsContainer); 
+    }
+
+    async isAlertMessageDisplayed(){
+        await this.amOnPage();
+        return await this.exuiAlert.isDisplayed(); 
+    }
+
+    async getAlertMessageText() {
+        if(await this.isAlertMessageDisplayed()){
+            return await this.exuiAlert.getText();
+        }else{
+            throw new error("Alert/notification  message is not displayed or disappeared.");
+        } 
+    }
+
+    async isCaseTitleDisplayed(){
+        await this.amOnPage();
+        return await this.caseTitle.isDisplayed();
+    }
+
+    async getCaseTitle() {
+        await this.amOnPage();
+        return await this.caseTitle.getText();
+    }
+
+    async isEventTriggerDisplayed(){
+        await this.amOnPage();
+        return await this.eventTriggerContainer.isDisplayed();
+    }
+
+    async GetEvents(){
+        const isTriggerElementDisplayed = this.isEventTriggerDisplayed();
+        if (!isTriggerElementDisplayed){
+            throw new Error("Event trigger element not displayed");
+        }
+        const eventsCount = await this.eventTriggerContainer.$$('option').count();
+        const events = [];
+        for (let eventCounter = 0; eventCounter < eventsCount; eventCounter++){
+            events.push(await this.eventTriggerContainer.$$('option')[eventCounter].getText());
+        }
+        return events;
+    }
+
+    async selectNextStepEvent(eventName){
+        await await this.eventTriggerContainer.$(`option[title = ${eventName}]`).click() 
```

Comment:
what does the doulbe await mean?

Reply:
@sreekanthpuligadda I believe this your change

Reply:
this file and couple of other files are nor required. They are MC specific files got in here as part copying framework from manage cases.